### PR TITLE
seeded species and botanist tables

### PR DESCRIPTION
**Pull request by jiuliangut**
#38 
- seed species table with lowercase plant_names 
- seed botanist table

### CODE REVIEW

File: extract.py [hunk 1]
```diff
@@ -78,4 +78,6 @@ def load_into_dataframe() -> pd.DataFrame:
 
 
 if __name__ == "__main__":
-    load_into_dataframe()
+    pd.set_option('display.max_rows', None)
+    pd.set_option('display.max_columns', None)
+    print(load_into_dataframe())
```
Comment (Fahi28) [inline extract.py]: no need to push the print statement

Comment (jiuliangut) [reply extract.py]: i keep accidentally pushing my local changes to extract - edited now!

**Review by Fahi28 — APPROVED** [review]
seeding looks good.

**Review by bentrzcinski — APPROVED** [review]
Nice seeding!

**Pull request merged**